concepts/Common/Licensing/20 Sharing Keys by Multiple Developers.md

```diff
@@ -0,0 +1,5 @@
+According to the EULA, every developer who works with DevExtreme must have a license, since the DevExtreme license has a per seat basis.
+
+If you have many licenses that you distribute among developers, then each developer will have access to their own personal key in Download Manager under their account.
+
+If a developer leaves or simply stops working with DevExtreme, then you can revoke their license in your personal account. This will not affect the license keys. They will still continue to work within the major versions for which they were released.
```

Comment:
```suggestion
If a developer leaves or simply stops working with DevExtreme, then you can revoke their license in your personal account. However, this will not affect the license keys that was already issued. it will remain to be valid for all updates under the major version to which it was bound (and the older ones)
```